tests/test_job.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from pybuildkite.jobs import Jobs
+from pybuildkite.client import Client
+from requests.exceptions import HTTPError
+
+class TestJob:
+
+    client = Client()
+    url = 'https://api.buildkite.com/v2/'
+
+    def test_Jobs(self) :
+
+        job = Jobs(self.client, self.url)
+        assert job.client == self.client
+        assert job.path == self.url+"/organizations/{}/pipelines/{}/builds/{}/jobs/{}/"
+
+    def test_get_job_log(self):
+
+        jobs = Jobs(self.client, self.url)
+        organization = 'TestOrganization'
+        pipeline = 'TestPipeline'
+        build = 'TestBuild'
+        job = 'TestJob'
+        with pytest.raises(HTTPError) as e:
+            jobs.get_job_log(organization, pipeline, build, job)
```

Comment:
Maybe we should look into `requests_mock` so that we aren't sending an actual http request in our unit tests?

Reply:
yes, you are right @pyasi if we use mock it's running time also get improved. 
thx let's fix these.

Reply:
There's an example of that over in #13 if it helps! 😁

Reply:
Thanks @PurpleBooth